jdj_tags/extensions.py

```diff
@@ -263,12 +264,12 @@ def _make_blocktrans(self, singular, plural=None, context=None, trans_vars=None,
         else:
             if context is None:
                 return ungettext(
-                    force_text(singular), force_text(plural), trans_vars[count_var]
+                    force_text(singular), force_text(plural), int(trans_vars[count_var])
```

Comment:
I'm not sure either, never dwelled much in how Jinja2 actually works. I thought you knew since your wrote this part of code! Or is it a trick question? :)